pkg/sysfs/system.go

```diff
@@ -1224,7 +1234,7 @@ func (sys *system) discoverNodes() error {
 				dramTotal += info.MemTotal
 			}
 		}
-		dramAvg = dramTotal / uint64(len(dramNodeIds))
+		dramAvg = dramTotal / uint64(len(dramNodeIds)-noMemNodes.Size())
```

Comment:
...need to be cautious with division by zero here. If there is a risk, let's error out with "unsupported memory configuration" rather than runtime error.

Reply:
Added an explicit check for a 0 divisor and bailout with an error if it happens.